examples/set-gen/main.go

```diff
@@ -35,6 +35,10 @@ import (
 
 func main() {
 	arguments := args.Default()
+
+	// Override defaults.
```

Comment:
why?

Reply:
@jennybuckley was this only for testing?

Reply:
Yes, this was to make the tests pass.

Reply:
should we fix the tests?

Reply:
imo, the tests are good how they are because they test if the behavior of the example generators have changed based on the changes to the generic gengo code. I could have run the generators and committed the changes to the generated code, instead of modifying the example generators, but that wouldn't show that the old behavior of the example generators can be implemented by setting the new generator argument, and the output will be exactly how it was before.

Reply:
Yes, please remove the overrides and update the test output in another commit.